src/Core/src/Platform/Android/IAndroidLifecycleHandler.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Android.App;
+using Android.Content;
+using Android.Content.Res;
+using Android.OS;
+using Android.Runtime;
+
+namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Allow to get Android Activity lifecycle callbacks.
+	/// </summary>
+	public interface IAndroidLifecycleHandler : IPlatformLifecycleHandler
```

Comment:
As mentioned over a few things and after a bit of awesome discussion... maybe this is more service-y than handler-y? At least as we have defined the words in MAUI.